docs/Configuration.md

```diff
@@ -63,3 +63,70 @@ Angulartics can be configured to work with a number of other services besides Go
 In order to start using one of these services, select it from the [Angulartics Providers page](https://angulartics.github.io/angulartics2/#providers), and follow the instructions on how to configure it.
 
 The Google Analytics script was added in [`main.browser.ts`](https://github.com/DSpace/dspace-angular/blob/ff04760f4af91ac3e7add5e7424a46cb2439e874/src/main.browser.ts#L33) instead of the `<head>` tag in `index.html` to ensure events get sent when the page is shown in a client's browser, and not when it's rendered on the universal server. Likely you'll want to do the same when adding a new service.
+
+## SEO when hosting REST Api and UI on different servers
+
+Indexers such as Google Scholar require that files are hosted on the same domain as the page that links them. In DSpace 7, Bitstreams are served from the REST server. So if you use different servers for the REST api and the UI you'll want to ensure that Bitstream downloads are proxied through the UI server. 
+
+In order to achieve this we'll need to do two things:
+- **Proxy the Bitstream downloads through the UI server.** You'll need to put a webserver such as httpd or nginx in front of the UI server in order to achieve this. [Below](#apache-http-server-config) you'll find a section explaining how to do it in httpd.
```

Comment:
The reason is both performance, and the ease with which it would be possible to tweak or add e.g. caching to this config. [That isn't supported out of the box for http-proxy-middleware](https://github.com/chimurai/http-proxy-middleware/issues/252) and would require quite a bit of JS code to add. 

For sitemaps it would likely also be a better idea to proxy them using httpd, that is something we should consider adding to our installation documentation.  But because sitemaps are relatively small files, and because every installation of dspace 7 will need it, we decided to make it available out of the box for people that want a simple setup.

A setup with a rest server and a UI on different hosts is more complicated by definition, so it is likely we can expect it won't have the express server for the UI or the tomcat for rest being the direct access points for clients

Reply:
@artlowel : Thanks for clarifying. I wanted to ensure this decision was documented clearly in this PR, which your comment now does.  I think this is perfectly OK for now, I just wanted us to clarify why file downloads are (at least currently) proxied differently from sitemaps....and I do agree that auto-proxying sitemaps is much more important here, as we want to ensure those are always enabled by default (for SEO purposes). I'm OK with the extra steps regarding file downloads if you choose a more complex installation (of different hosts for UI vs REST)